src/msw/webview_edge.cpp

```diff
@@ -246,6 +246,23 @@ class wxWebViewEdgeHandlerResponse : public wxWebViewHandlerResponse
     wxCOMPtr<ICoreWebView2WebResourceRequestedEventArgs> m_args;
 };
 
+class wxWebViewImplCallbackActivator
```

Comment:
I wonder if we should add some `wxScopedVarSetter` that could be used as
```
wxScopedVarSetter set(m_impl->m_inEventCallback, true);
```
as it looks like we do something like this in many places in our code.

There is also a `wxRecursionGuard`, but it's not quite the same thing...

Reply:
While looking where to add something like `wxScopedVarSetter` I've came accross `wxON_BLOCK_EXIT_SET()` which seems perfectly suitable for this scenario and I've used that now.